library/std/src/sys/unix/ext/process.rs

```diff
@@ -31,6 +31,11 @@ pub trait CommandExt {
         #[cfg(target_os = "vxworks")] id: u16,
     ) -> &mut process::Command;
 
+    /// Sets the supplementary group IDs for the calling process. Translates to
+    /// a `setgroups` call in the child process.
+    #[unstable(feature = "setgroups", issue = "38527", reason = "")]
+    fn groups(&mut self, groups: &[u32]) -> &mut process::Command;
```

Comment:
Does this also need to be u16 on vxworks?

Reply:
I'm not too sure about vxworks. Maybe the other reviewers can comment on this @joshtriplett

Reply:
Nvm I see what you mean. gid() and uid() are taking in u16 for vxworks